test/__mocks__/text.mock.ts

```diff
@@ -254,6 +254,7 @@ export const REMOVE_SOLD_ALL_LAND_FILTER_PAGE_TITLE = "Has the overseas entity d
 export const REMOVE_IS_ENTITY_REGISTERED_OWNER_TITLE = "Is the overseas entity currently listed on any land registry records as the registered owner of property or land in the UK?";
 export const REMOVE_INTERRUPT_CARD_TEXT = "Once an overseas entity is removed";
 export const REMOVE_USE_INFORMATION_NEED_MORE = "We’ll use this if we need more information about the application.";
+export const REMOVE_CONFIRMATION_STATEMENT_TITLE = "Confirm the removal statement - Apply to remove and overseas entity";
```

Comment:
Text content here doesn't look right - 'and overseas entity'. Surely it should be 'an overseas entity'?

But if this is wrong and tests are passing, either it's not actually being used by a test OR it's also wrong somewhere else.